examples/gno.land/r/sys/vals/vals.gno

```diff
@@ -0,0 +1,83 @@
+package vals
+
+import (
+	"std"
+
+	"gno.land/p/demo/json"
```

Comment:
Do you suggest we simply return the typed set itself, instead of marshaling it to JSON?
The biggest concern I have there is that clients will be unable to parse the returned set, because our VM calls return values that need custom parsing (and don't contain type info)